stripe-core/src/main/AndroidManifest.xml

```diff
@@ -1,2 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest />
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
```

Comment:
This shouldn’t resolve in issues for integrators.

Reply:
Did you mean `This shouldn’t result in issues for integrators.`

Reply:
I did!